WSSiOS/WSSiOS/Resource/Constants/Strings/StringLiterals+Feed.swift

```diff
@@ -46,6 +46,13 @@ extension StringLiterals {
         static let spoilerText = "스포일러가 포함된 글 보기"
         static let modifiedText = "(수정됨)"
         static let isPrivate = "나만 보는 기록이에요."
+        
+        enum Filter {
+            static let title = "글 찾기 필터"
+            static let genre = "장르"
+            static let visibliltyOption = "공개범위"
```

Comment:
```suggestion
            static let visibliltyOption = "공개 여부"
```
라고 피그마상에선 나와있는데 수정된 부분 일까여?

Reply:
헛 공개여부 네요 수정하겠습니다